.babelrc

```diff
@@ -8,5 +8,7 @@
       "react-hot-loader/babel",
       "transform-object-rest-spread",
       "transform-class-properties",
+      "transform-react-constant-elements",
```

Comment:
I think so - we can tell the `babel-loader` webpack plugin to use certain babel plugins (plugins within plugins...)

I had read that in the documentation too, I'll try to get that set up now.